slides/r-adoption-jan2023.qmd

```diff
@@ -0,0 +1,204 @@
+---
+title: "MMRM and ASA SWE - R Adoption Series"
```

Comment:
so I was actually thinking to have a single file in this branch first where we all work on the slides together - and then you can still tweak your slides later, what do you think? since currently there are still 2 files - one for r govys and one for r adoption